src/solvers/changeCobraSolverParams.m

```diff
@@ -56,9 +56,9 @@
 allowedQPparams = {'minNorm', 'printLevel'};
 allowedMILPparams = {'intTol','relMipGapTol','timeLimit','logFile','printLevel'};
 
-% Only LP is currently included
+% Only LP, QP and MILP are currently included

```

Comment:
Just as a general question,
Wouldn't it make sense to change these parameters from global variables to persistent variables in this function? 
This would force usage of this function for parameter changes, and thus avoid tracking issues.
Alternatively an actual config file could be used, which could actually be accessed by paralell workers (but that would make it much slower probably)